doc/private_endpoint_connection_command_guideline.md

```diff
@@ -0,0 +1,84 @@
+## Command Guideline
+
+#### Private Endpoint Connection
+
+- The parent resource should expose a single command group called `private-endpoint-connection` with four commands: `approve`, `reject`, `delete`, `show`.
+- If `approve` and `reject` are long running operations, please also provide `... private-endpoint-connection wait` command and support `--no-wait` in `approve` and `reject` commands.
+- The `... private-endpoint-connection approve` command should look similar to the following, depending on which features are supported by the service.
+```
+Arguments
+    --approval-description          : Comments for the approval.
```

Comment:
add required/optional property

Reply:
this argument is optional. If it's not marked as required, it's optional.